pom.xml

```diff
@@ -595,6 +594,13 @@ under the License.
 				<version>${junit4.version}</version>
 			</dependency>
 
+			<dependency>
+				<groupId>org.assertj</groupId>
+				<artifactId>assertj-core</artifactId>
+				<version>${assertj.version}</version>
+				<scope>test</scope>
+			</dependency>
```

Comment:
Nit: Should we make a separate PR to add this change because it is currently necessary to use assertj in any other module?

Reply:
Our dependencies already include assertj: https://github.com/apache/flink/commit/c4b604885a98aead3bf109189cbff7d5d95ade72. This change adds it to the `dependencyManagement` as well in order to let flink-test-utils-junit pick the right version